src/screens/shared/UserProfile.js

```diff
@@ -222,40 +220,28 @@ class UserProfile extends React.Component {
               <form>
                 <div>
                   <p>
-                    <label htmlFor="updateFirstName">
-                      First Name
-                      {this.renderInputLabel(
-                        'updateFirstName',
-                        generalEditMode
-                      )}
+                    <label htmlFor="updateName">
+                      Name
+                      {this.renderInputLabel('updateName', generalEditMode)}
                     </label>
                   </p>
                 </div>
                 <div>
                   <p>
-                    <label htmlFor="updateLastName">
-                      Last Name
-                      {this.renderInputLabel('updateLastName', generalEditMode)}
+                    <label htmlFor="updateEmail">
```

Comment:
is there a reason for this label tag? in non-edit mode this is not a form.

Reply:
change to p tags? make sure they're in rows